src/config/mod.rs

```diff
@@ -249,6 +253,12 @@ impl VisualBellConfig {
     pub fn color(&self) -> Rgb {
         self.color
     }
+
+    /// Visual bell command
+    #[inline]
+    pub fn bell_command(&self) -> Option<CommandWrapper> {
+        self.bell_command.clone()
+    }
```

Comment:
We don's have getters for the other launcher (it's simply a `pub` field). I don't love the idea of having a lone `pub` field on the struct, but at the same time, I really would like to avoid `clone`. Thoughts?

Reply:
Using `.as_ref()` would probably work here to return an `Option<&CommandWrapper>`. No clones necessary.

Reply:
I'm not quite sure how to do this? surely you don't want me to litter all the structures with lifetime annotations just because of this?

Reply:
@robertgzr like this:

```rust
struct A(Option<u32>);

impl A {
    fn get(&self) -> Option<&u32> {
        self.0.as_ref()
    }
}

fn main() {
    println!("{:?}", A(Some(10)).get());
    println!("{:?}", A(None).get());
}
```